pkg/controller/buildrun/buildrun_controller.go

```diff
@@ -299,10 +298,14 @@ func (r *ReconcileBuildRun) retrieveTaskRun(build *buildv1alpha1.Build, buildRun
 	return nil, nil
 }
 
-func (r *ReconcileBuildRun) updateBuildRunErrorStatus(buildRun *buildv1alpha1.BuildRun, err error) *buildv1alpha1.BuildRun {
+func (r *ReconcileBuildRun) updateBuildRunErrorStatus(buildRun *buildv1alpha1.BuildRun, err error) error {
 	buildRun.Status.Succeeded = corev1.ConditionFalse
 	buildRun.Status.Reason = err.Error()
 	now := metav1.Now()
 	buildRun.Status.StartTime = &now
-	return buildRun
+	updateErr := r.client.Status().Update(context.TODO(), buildRun)
```

Comment:
this should be a one-liner

Reply:
Yep, I thought about that when I developing. But I see other logics still separate as two lines. So I keep the same style as before.

If in future, we would like to use one line, let us refine together.

Reply:
Im not sure if this is a good argument, if you have room for improving the code for a one liner, you should always do this.

Reply:
Do you mean these two lines?:
```
	now := metav1.Now()
	buildRun.Status.StartTime = &now
```

If yes, I tried to make two lines to one line before, but it doesn't work:
`buildRun.Status.StartTime = &metav1.Now()` reports error `Cannot take the address of metav1.Now()`, that is why I separate it to two lines

Reply:
no, I meant the client status update call.

Reply:
I am not sure if it is already a standard in the community (Where can I see this kind of standard?) or if we have to do that in this PR.

At least, I reviewed several reconcilers in different communities, they allALL do by two lines, to me, it is more clear, like:
```
memcached := &cachev1alpha1.Memcached{}
err := r.client.Get(context.TODO(), request.NamespacedName, memcached)
if err != nil {
...
}
```

- Tekton pipeline: https://github.com/tektoncd/pipeline/blob/master/pkg/reconciler/taskrun/taskrun.go#L83-L88

- Knative serving: https://github.com/knative/serving/blob/master/pkg/reconciler/service/service.go#L104-L107

- operator-sdk user guide: https://github.com/operator-framework/operator-sdk/blob/master/doc/user-guide.md

- kpack: https://github.com/pivotal/kpack/blob/master/pkg/reconciler/v1alpha1/build/build.go#L104-L107

So I suggest we still do in this PR until we all agree or confirm our common format or code style.

Reply:
leaving this for a broader audience also. This clearly can be a one-liner

Reply:
i spoke with Jordan offline, we agree this should not block the PR, so Im resolving the request for changes